spacy/matcher/matcher.pyx

```diff
@@ -796,9 +798,14 @@ def _get_attr_values(spec, string_store):
                 attr = "ORTH"
             if attr == "IS_SENT_START":
                 attr = "SENT_START"
+            if attr == "ENT_IOB":
+                ent_iob = True
             attr = IDS.get(attr)
         if isinstance(value, str):
-            value = string_store.add(value)
+            if orig_attr == "ENT_IOB" and value in iob_strings:
```

Comment:
```suggestion
            if attr == ENT_IOB and value in Token.iob_strings():
```

Just use the attr `ENT_IOB` and `Token.iob_strings()` directly here, I don't think there's a real need to store them in additional variables. If it were code that got run while matching, perhaps, but this is just called when the pattern is added and most people won't be using `ENT_IOB` at all, anyway.

Reply:
1. OK, fair point. I killed `self._ent_iob-strings`
2.  `attr` turns into an integer after processing it. By the line where we process the `value` , it's an ID already:

```
attr = IDS.get(attr)
        
if isinstance(value, str):
```

That's why I restored the original attribute string.

Reply:
You want to add an import for the attr `ENT_IOB` here:

https://github.com/explosion/spaCy/blob/24cdd4c88e291d01aff9cd92c37c6cdb03479729/spacy/matcher/matcher.pyx#L21

Reply:
Oh kapish, I do it